cmd/azure-config-credentials-injector/main.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	clientIDEnvKey     = "AZURE_CLIENT_ID"
+	clientSecretEnvKey = "AZURE_CLIENT_SECRET"
+
+	clientIDCloudConfigKey     = "aadClientId"
+	clientSecretCloudConfigKey = "aadClientSecret"
+)
+
+func main() {
+	rootCmd := &cobra.Command{
+		Use:   "azure-config-credentials-injector [cloud-config-file-path] [output-file-path]",
+		Short: "Cloud config credentials injection tool for azure cloud platform",
+		Args:  cobra.ExactArgs(2),
```

Comment:
Wondering if it's worth using named arguments rather than positional arguments, any thoughts?